build.gradle

```diff
@@ -202,12 +202,15 @@ project('spring-rabbit') {
 		compile project(":spring-amqp")
 
 		compile "com.rabbitmq:amqp-client:$rabbitmqVersion"
-		compile ("com.rabbitmq:http-client:$rabbitmqHttpClientVersion", optional)
+		compile ("com.rabbitmq:http-client:$rabbitmqHttpClientVersion") {
+			exclude group: 'org.springframework'
+		}
```

Comment:
My biggest concern is that it has a dependency on `spring-web` and we don't want mixed spring versions on the CP.